tests/unit/source_code/test_site_packages.py

```diff
@@ -1,7 +1,27 @@
-from databricks.labs.ucx.source_code.site_packages import SitePackages
+from databricks.labs.ucx.source_code.files import FileLoader
+from databricks.labs.ucx.source_code.graph import DependencyProblem
+from databricks.labs.ucx.source_code.site_packages import PipResolver, SitePackages
 from tests.unit import locate_site_packages
 
 
+def test_pip_resolver_resolve_library(mock_path_lookup):
```

Comment:
Tests require internet access, maybe should be a integration test because of that